packaging/bundle-validation/validate.sh

```diff
@@ -280,18 +280,12 @@ if [ "$?" -ne 0 ]; then
 fi
 echo "::warning::validate.sh done validating spark & hadoop-mr bundle"
 
-#TODO [HUDI-8295] get this working for some of the supported spark versions
-if [[ $SPARK_HOME == *"spark-2.4"* ]] || [[  $SPARK_HOME == *"spark-3.1"* ]]
-then
-  echo "::warning::validate.sh validating utilities bundle"
-  test_utilities_bundle $JARS_DIR/utilities.jar
-  if [ "$?" -ne 0 ]; then
-      exit 1
-  fi
-  echo "::warning::validate.sh done validating utilities bundle"
-else
-  echo "::warning::validate.sh skip validating utilities bundle for non-spark2.4 & non-spark3.1 build"
+echo "::warning::validate.sh validating utilities bundle"
+test_utilities_bundle $JARS_DIR/utilities.jar
```

Comment:
In the following, we do test on utilities-slim.jar. So we should just remove this check since spark <= 3.1 is not supported.

Reply:
We want to test that the non-slim bundle isn't missing anything

Reply:
since spark < 3.1 is not maintained, should we remove this bundle as well?

Reply:
nope. we generate the fat utilities bundle only for 1 version of spark2 and 1 version of spark3. 

for eg, 
we have 3 spark bundles for spark pertaining to spark3.1, spark3.2, spark3.4 and spark3.5 
but for utilities, we only have 1 bundle. 

for diff combinations of spark versions, we recommend users to go w/ utilities slim bundle + the resp spark bundle.